services/w3c/w3c-validation-helper.spec.js

```diff
@@ -0,0 +1,321 @@
+'use strict'
+const { expect } = require('chai')
+const {
+  presetRegex,
+  getMessage,
+  getColor,
+  getSchema,
+} = require('./w3c-validation-helper')
+
```

Comment:
The tests look good as is, but you may want to also check out `sazerac` which we use elsewhere. It basically helps you get to the same endstate without having to define a lot of the boilerplate yourself. 

For example:
https://github.com/badges/shields/blob/master/core/base-service/auth-helper.spec.js#L19-L45
https://github.com/badges/shields/blob/4c159eff004f78b176ae72bb20337d9e95757074/core/base-service/auth-helper.spec.js#L19-L45

and
https://github.com/badges/shields/blob/master/services/jenkins/jenkins-build.spec.js
https://github.com/badges/shields/blob/4c159eff004f78b176ae72bb20337d9e95757074/services/jenkins/jenkins-build.spec.js#L11-L40

Reply:
Replace forEach tests with `sazerac`